app/controllers/runs/likes_controller.rb

```diff
@@ -25,12 +25,12 @@ def destroy
   private
 
   def set_run
-    @run = Run.find_by(id: params[:run].to_i(36))
+    @run = Run.find_by(id: params[:run_id].to_i(36))
     head :not_found if @run.nil?
   end
 
   def set_run_like
-    @run_like = RunLike.find_by(user: current_user, run: Run.find_by(id: params[:run].to_i(36)))
+    @run_like = RunLike.find_by(user: current_user, run: Run.find_by(id: params[:run_id].to_i(36)))
```

Comment:
I made these changes because I've been wanting to revert us back to conventional Rails style for `params` for a while, and the rest of this branch got easier if I went ahead and did parts of it now.

Reply:
Is there a reason you can't do this here:
```
Run.find(params[:run_id].to_i(36))
```

Reply:
`.find` throws an error vs `.find_by` returning nil on no record found.